splinterd/src/config/mod.rs

```diff
@@ -287,8 +296,24 @@ impl Config {
         }
     }
 
+    #[allow(clippy::cognitive_complexity)]
     /// Displays the configuration value along with where the value was sourced from.
     pub fn log_as_debug(&self) {
+        debug!(
+            "Config: config_dir: {:?} (source: {:?})",
+            self.config_dir(),
+            self.config_dir_source()
+        );
+        debug!(
```

Comment:
These were previously sorted alphabetically. That is probably still the best approach here?

Reply:
I don't think these were previously sorted alphabetically, it seems like they've always been sorted in the order the entry is in the Config object. I've added the log for the config_dir first, as this is now the first entry in the Config objects.